i18n/es.json

```diff
@@ -1192,6 +1192,14 @@
     "id": "api.file.upload_file.incorrect_number_of_files.app_error",
     "translation": "No se pudo subir los archivos. El número de archivos especificado es incorrecto."
   },
+  {
+    "id": "api.file.upload_file.incorrect_channelId.app_error",
+    "translation": "No se pueden cargar archivos. Id. De canal incorrecto: {{.channelId}}"
+  },
+  {
+    "id": "api.file.upload_file.user_not_authorize_upload.app_error",
+    "translation": "No se pueden cargar archivos. Se requiere permiso de carga."
```

Comment:
And the same here: `No tienes los permisos necesarios para subir un archivo al canal con identificador {{.channelId}}.`